wire-ios-request-strategy/Sources/Request Strategies/Connection/ConnectionRequestStrategy.swift

```diff
@@ -195,10 +195,12 @@ extension ConnectionRequestStrategy: ZMEventConsumer {
             case .userConnection:
                 if let conversationEvent = Payload.UserConnectionEvent(payloadData) {
                     let processor = ConnectionPayloadProcessor()
-                    processor.processPayload(
-                        conversationEvent,
-                        in: managedObjectContext
-                    )
+                    managedObjectContext.performAndWait {
```

Comment:
why is this `performAndWait` necessary?  The non-async`processEvents` is should always be executed on the sync context?